common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -629,78 +629,124 @@ def test_navigation_links(self):
     def verify_and_navigate_to_create_team_page(self):
         """Navigates to the create team page and verifies."""
         self.browse_teams_page.click_create_team_link()
-        self.verify_page_header()
+        self.verify_page_header(
+            title='Create a New Team',
+            description='Create a new team if you can\'t find existing teams to '
+                        'join, or if you would like to learn with friends you know.',
+            breadcrumbs=self.topic['name']
+        )
+
+    def verify_and_navigate_to_edit_team_page(self):
+        """Navigates to the edit team page and verifies."""
+        # pylint: disable=no-member
+        self.assertEqual(self.team_page.team_name, self.team['name'])
+        self.assertTrue(self.team_page.edit_team_button_present)
+
+        self.team_page.click_edit_team_button()
+
+        self.create_or_edit_team_page.wait_for_page()
 
-    def fill_create_form(self):
-        """Fill the create team form fields with appropriate values."""
-        self.create_team_page.value_for_text_field(field_id='name', value=self.team_name, press_enter=False)
-        self.create_team_page.value_for_textarea_field(
+        # Edit page header.
+        self.verify_page_header(
+            title='Edit Team',
+            description='If you make significant changes, make sure you notify '
+                        'members of the team before making these changes.',
+            breadcrumbs='All Topics {topic_name} {team_name}'.format(
+                topic_name=self.topic['name'],
+                team_name=self.team['name']
+            )
+        )
+
+    def verify_team_info(self, name, description, location, language):
+        """Verify the team information on team page."""
+        # pylint: disable=no-member
+        self.assertEqual(self.team_page.team_name, name)
+        self.assertEqual(self.team_page.team_description, description)
+        self.assertEqual(self.team_page.team_location, location)
+        self.assertEqual(self.team_page.team_language, language)
+
+    def fill_create_or_edit_form(self):
+        """Fill the create/edit team form fields with appropriate values."""
+        self.create_or_edit_team_page.value_for_text_field(field_id='name', value=self.team_name, press_enter=False)
+        self.create_or_edit_team_page.value_for_textarea_field(
             field_id='description',
-            value='The Avengers are a fictional team of superheroes.'
+            value=self.TEAM_DESCRIPTION
         )
-        self.create_team_page.value_for_dropdown_field(field_id='language', value='English')
-        self.create_team_page.value_for_dropdown_field(field_id='country', value='Pakistan')
+        self.create_or_edit_team_page.value_for_dropdown_field(field_id='language', value='English')
+        self.create_or_edit_team_page.value_for_dropdown_field(field_id='country', value='Pakistan')
 
-    def test_user_can_see_create_team_page(self):
+    def verify_team_data(self):
```

Comment:
This is about the verifying that the fields are present with the right help messages. Can we make this `verify_all_fields_exist`?